src/vector/defn.rs

```diff
@@ -66,4 +74,16 @@ impl<'a> Field<'a> {
         let rv = unsafe { ogr::OGR_Fld_GetNameRef(self.c_field_defn) };
         return _string(rv);
     }
+
+    pub fn get_type(&'a self) -> i32 {
```

Comment:
I guess I can't use `type` as a function name as it is keyword in rust (that's why I chose `get_type` in replacement and thus to be consistent over the methods of this object I also used `get_width` and `get_precision`). Any suggestions ?